### PR TITLE
Animal table

### PR DESCRIPTION
**Pull request by AlexRS90**
## For this milestone I:
✅  Created a new database.
✅  Added a new table.
✅  Inserted some values.
✅  Wrote some queries.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE animal_name LIKE '%mon%';
```
Comment (SajjadAhmad14) [inline queries.sql]: Please write a query to find all animals that have names ending with "mon", not with names starting and ending with "mon".

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+SELECT * FROM animals WHERE date_of_bird >= '01-01-2016'::date AND date_of_bird <= '12-31-2019'::date;
+SELECT * FROM animals WHERE neutered IN (true) AND escape_attempts < 3;
+SELECT date_of_bird FROM animals WHERE animal_name IN ('Agumon', 'Pikachu');
```
Comment (SajjadAhmad14) [inline queries.sql]: Please correct the typo here as it is `date_of_birth` not `date_of_bird`.

File: data.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
```
Comment (SajjadAhmad14) [inline data.sql]: Please add table attributes before inserting their values as this ` INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg)`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Alex, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional.  :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (AlexRS90) [reply queries.sql]: Thanks, I'll work on it!

File: queries.sql (hunk 2 above)
Comment (AlexRS90) [reply queries.sql]: Ops 😖 , my bad, thanks for your review!!

File: data.sql (hunk 3 above)
Comment (AlexRS90) [reply data.sql]: I did it, but just a question, is this a good practice?

**Review by descholar-ceo — APPROVED** [review]
Hi @AlexRS90, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**